### PR TITLE
Issue/2794 dashboard visualization error

### PR DESCRIPTION
**Pull request by melowsoft**
- [ ] **Update release notes if necessary**

- Added Error screen for dashbaord error
- Fixed undefined error

### CODE REVIEW

File: client/src/actions/collection.js [hunk 1]
```diff
@@ -73,16 +73,29 @@ export function deleteCollection(id) {
 export function addEntitiesToCollection(entityIds, collectionId) {
   return (dispatch, getState) => {
     const collection = getState().library.collections[collectionId];
+   
+    console.log(entityIds, collection, "for need purpose")
     const visualisations = uniq([...entityIds.visualisations, ...collection.visualisations]);
+    console.log( "i got here finally, where the bug is")
```
Comment (iperdomo) [inline client/src/actions/collection.js:79]: Please remove those debugging logging

**Review by iperdomo — CHANGES_REQUESTED** [review]
I think this branch has a mixture of 2 different problems:

* Adding a dataset within an existing collection
* Showing an error message when the dashboard doesn't exist

Is this case? 🤔 If so, let's split the changes in 2 branches, one for the dataset problem, other for the visualization error

**Comment by iperdomo** [comment]
I would also suggest to fix your committer settings, right now is using a non recognized email address.

https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address

**Comment by iperdomo** [comment]
@melowsoft this branch still has changes that are *not* related to the issue. Can you update/fix this branch so we have only the necessary changes for the issue at hand?

**Comment by melowsoft** [comment]
> @melowsoft this branch still has changes that are _not_ related to the issue. Can you update/fix this branch so we have only the necessary changes for the issue at hand?

okay, I will fix this

**Pull request closed without merging**